docs/reference/search/request/script-fields.asciidoc

```diff
@@ -31,7 +31,8 @@ GET /_search
 }
 --------------------------------------------------
 // CONSOLE
-
+// TEST[setup:sales]
+// TEST[s/my_field_name/price/]
```

Comment:
I'd prefer to change the example so the test lines up a little closer. it isn't a big deal though.